src/distributions/microsoft/installer.ts

```diff
@@ -0,0 +1,122 @@
+import { JavaBase } from '../base-installer';
+import { JavaDownloadRelease, JavaInstallerOptions, JavaInstallerResults } from '../base-models';
+import semver from 'semver';
+import { extractJdkFile, getDownloadArchiveExtension, isVersionSatisfies } from '../../util';
+import * as core from '@actions/core';
+import { MicrosoftVersion, PlatformOptions } from './models';
+import * as tc from '@actions/tool-cache';
+import fs from 'fs';
+import path from 'path';
+
+const supportedPlatform = `'linux', 'macos', 'windows'`;
```

Comment:
The URL in my browser shows `macOS`, but I see `macos` works as well (IIS? 🙂)

![image](https://user-images.githubusercontent.com/33549821/143943594-46755a73-24b3-45bd-992e-fdb64a052b08.png)

Reply:
I don't think there's anything needed here. (Aren't all URLs case-insensitive?)

Reply:
I believe the path part of a URL is usually case-sensitive: https://webmasters.stackexchange.com/questions/90339/why-are-urls-case-sensitive

It seems to work though, so no not a big deal.